event-utils.html

```diff
@@ -0,0 +1,123 @@
+<!--
+    @license
+    Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+
+`events-helper` is a mixin containing helper functions to work with events.
+
+@group Polymer Mixins
+@element Polymer.Core.EventUtils
+@homepage github.io
+
+-->
+
+<script>
+
+  Polymer.Core = Polymer.Core || {};
+
+  /* TODO: Move to own repo */
+
+  Polymer.Core.EventUtils = {
+
+    /**
+     * Listens for a set of events on a node.
+     *
+     * @method addElementListenerList
+     * @param {Node} node
+     * @param {Object} events eventname,methodname pairs
+     */
+    addElementListenerList: function(node, events) {
+      for (var i in events) {
+        this.addElementListener(node, i, events[i]);
+      }
+    },
+
+    /**
+     * Removes a set of event listeners from a node.
+     *
+     * @method removeElementListenerList
+     * @param {Node} node
+     * @param {Object} events eventname,methodname pairs
+     */
+    removeElementListenerList: function(node, events) {
+      for (var i in events) {
+        this.removeElementListener(node, i, events[i]);
+      }
+    },
+
+    /**
+     * Toggles an event listener on a node.
+     *
+     * @method enableElementListener
+     * @param {Boolean} enable
+     * @param {Node} node
+     * @param {String} event
+     * @param {String} methodName
+     * @param {Boolean} capture
+     */
+    enableElementListener: function(enable, node, event, methodName, capture) {
+      if (enable) {
+        this.addElementListener(node, event, methodName, capture);
+      } else {
+        this.removeElementListener(node, event, methodName, capture);
+      }
+    },
+
+    /**
+     * Add an event listener on a node.
+     *
+     * @method addElementListener
+     * @param {Node} node
+     * @param {String} event
+     * @param {String} methodName
+     * @param {Boolean} capture
+     */
+    addElementListener: function(node, event, methodName, capture) {
```

Comment:
I'm still a little confused why `listeners` isn't sufficient for this :confused: